desktop/app/splash.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+// Checks to see if we've launched before (we touch started.txt). If not, write the file and open a browser
+
+const helpURL = 'https://keybase.io/docs/cli_kbstage'
+const filePath = 'started.txt'
```

Comment:
This makes the path pretty non-deterministic, I think -- it's the directory you happened to be inside when you ran Electron?  What's going to happen when we launch via double clicking on an icon in Applications etc?

Using the same dir we put the keybase.leveldb in or similar sounds good to me?